package.xml

```diff
@@ -4,8 +4,8 @@
   <name>ros1_bridge</name>
   <version>0.10.2</version>
   <description>A simple bridge between ROS 1 and ROS 2</description>
-  <maintainer email="jacob@openrobotics.org">Jacob Perron</maintainer>
```

Comment:
Same comment as https://github.com/ros2/tinyxml_vendor/pull/25#discussion_r744710444

Reply:
Done in 972b77a.